examples/QIR/Optimization/README.md

```diff
@@ -0,0 +1,274 @@
+# QIR Generation and Optimization
+
+Since QIR is a specification of LLVM IR, the IR code can be manipulated via the usual tools provided by LLVM, such as [Clang](https://clang.llvm.org/), the [LLVM optimizer](https://llvm.org/docs/CommandGuide/opt.html), the [LLVM static compiler](https://llvm.org/docs/CommandGuide/llc.html), and the [just-in-time (JIT) compiler](https://llvm.org/docs/CommandGuide/lli.html).
+This example is structured as a walk-through of the process covering installation, QIR generation, and running built-in and custom LLVM optimization pipelines.
+
+## Prerequisites
+
+QIR generation is handled by the Q# compiler, while optimizations are performed at the LLVM level.
+The following software will be used in this walk-through:
+
+* Quantum Development Kit (QDK), containing the Q# compiler 
+* Clang, an LLVM based compiler for the C language family
+* LLVM optimizer, a tool to run custom optimization pipelines on LLVM IR code
+
+### Installing the QDK
+
+The *Quantum Development Kit (QDK)* documentation provides detailed guides on [installing the QDK](https://docs.microsoft.com/en-us/azure/quantum/install-overview-qdk#install-the-qdk-for-quantum-computing-locally) for various setups.
```

Comment:
Having `en-us` in URLs overrides the user's language preferences, such that best practice is to remove from all docs links.

```suggestion
The *Quantum Development Kit (QDK)* documentation provides detailed guides on [installing the QDK](https://docs.microsoft.com/azure/quantum/install-overview-qdk#install-the-qdk-for-quantum-computing-locally) for various setups.
```

Reply:
Good catch, I normally strip these but it seems I missed this one.